packages/web-pkg/src/composables/keyboardActions/useKeyboardActions.ts

```diff
@@ -65,12 +65,12 @@ const areCustomKeyBindingsDisabled = () => {
   return isTextSelected
 }
 
-export const useKeyboardActions = (): KeyboardActions => {
+export const useKeyboardActions = (skipDisabledKeyBindingsCheck?: boolean): KeyboardActions => {
```

Comment:
I'd prefer passing an object here. It's easier to handle with booleans since you're not only passing `true` or `false` into the function then.